examples/knative/powershell/kn-ps-email/Dockerfile

```diff
@@ -5,14 +5,15 @@ ENV PORT 8080
 # Set terminal. If we don't do this, weird readline things happen.
 RUN echo "/usr/bin/pwsh" >> /etc/shells && \
     echo "/bin/pwsh" >> /etc/shells && \
-    tdnf install -y powershell-7.0.3-2.ph3 unzip && \
+    tdnf install -y powershell icu openssl && \
     pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted" && \
-    find / -name "net45" | xargs rm -rf && \
-    tdnf erase -y unzip && \
-    tdnf clean all
-RUN pwsh -c 'Install-Module ThreadJob -Force -Confirm:$false -RequiredVersion 2.0.3' && \
+    pwsh -c 'Install-Module ThreadJob -Force -Confirm:$false -RequiredVersion 2.0.3' && \
     pwsh -c 'Install-Module -Name CloudEvents.Sdk -RequiredVersion 0.2.0' && \
     pwsh -c 'Install-Module -Name Send-MailKitMessage -RequiredVersion 3.1.0'
+    find / -name "net45" | xargs rm -rf && \
```

Comment:
What is this doing?

Reply:
Per Dimitar - This removes windows only content, which reduces the size of the image by almost half! This was something I didn't know but saw that we did with the official PowerCLI Docker Image